### DISCUSSION TITLE
下个版本能否增加cernet源

### DISCUSSION THREAD

**Discussion by 8MiYile** [istoreos/istoreos#1370] [Q&A] [answered]
https://help.mirrors.cernet.edu.cn/openwrt/

**Comment by jjm2473** [comment]
OK，如果速度快的话可以加上。

**Reply by 8MiYile** [reply]
因为这个原理是自动302
这个cernet是大学源聚合 他根据IP分配到最近的源
我们这个就是指路由器设备的IP
然后再传输
这样的话 每个源都会被用到

**Reply by jjm2473** [reply]
嗯，我测试了，会自动302到合适的源，不过，我看cernet列表里面的某些源（例如哈工大 https://mirrors.hit.edu.cn/openwrt ）我访问不了，可能只有教育网才能访问，所以这个302跳转不知道是不是可靠。下个版本默认切换到 cernet 源，看用户反馈吧，反正有问题可以选其他源。

**Reply by 8MiYile** [reply]
那就加进去 然后把其他大学源的移除掉 反正固件内还有华为源之类的

**Reply by 8MiYile** [reply]
查到了 哈工大那个IP是教育网的 v4和v6都一样是教育网的 但是实际工作是不会被分配到那里的 cernet会判断用户的IP地址

**Reply by jjm2473** [reply]
不知道cernet的302逻辑有没有区分教育网，还是只是靠geoip区分地区。除非有代码或维护者能证明有这个逻辑，不然还是保留以前的源，方便出问题时切换。

**Reply by ZhiShengYuan** [reply]
mirrorz是基于GEOIP来进行302的
对于HIT这种自行选择禁止校外访问的mirrorz能够进行正确的处理

**Reply by ZhiShengYuan** [reply]
https://github.com/mirrorz-org/mirrorz-d-extension/blob/master/custom/static/hit.json

**Reply by ZhiShengYuan** [reply]
update:沟通了一下，HIT暂时不会被重定向到了

**Reply by jjm2473** [reply, marked as answer]
下个版本固件会自动切换到cernet，但是为了在cernet故障时能切换到其他源，之前的大学开源镜像源还是保留着。
